gst/mqtt/mqttsink.c

```diff
@@ -932,7 +932,7 @@ gst_mqtt_sink_set_caps (GstBaseSink * basesink, GstCaps * caps)
     char *caps_str = gst_caps_to_string (caps);
```

Comment:
Q: What happens if using the returned ```gchar *``` for ```gst_caps_str ```
```suggestion
    self->mqtt_msg_hdr.gst_caps_str = gst_caps_to_string (caps);
```

Reply:
In my opinion, it is better to limit the size for safe string copying.